mediatype/mediatype.go

```diff
@@ -38,8 +42,13 @@ const (
 //   - Unquoted values in media parameters containing 'tspecials' characters
 //   - Newline characters
 func Parse(ctype string) (mtype string, params map[string]string, invalidParams []string, err error) {
+	return ParseWithOptions(ctype, MediaTypeParseOptions{})
+}
+
+// Parse method with additional options controlling the parsing behavior
```

Comment:
```suggestion
// ParseWithOptions parses media-type with additional options controlling the parsing behavior.
```

our godoc linter is broken right now